lib/evaluate.js

```diff
@@ -25,16 +27,26 @@ export const evaluate = async ({
 
   // Calculate reward shares
   const participants = {}
+  let sum = 0n
   for (const measurement of honestMeasurements) {
     if (!participants[measurement.participantAddress]) {
       participants[measurement.participantAddress] = 0n
     }
     participants[measurement.participantAddress] += 1n
   }
   for (const [participantAddress, participantTotal] of Object.entries(participants)) {
-    participants[participantAddress] = participantTotal *
-      1_000_000_000_000_000n /
+    const score = participantTotal *
+      MAX_SCORE /
       BigInt(honestMeasurements.length)
+    participants[participantAddress] = score
+    sum += score
+  }
+
+  if (Object.keys(participants).length && sum < MAX_SCORE) {
```

Comment:
Discussion point: how do we want to handle empty rounds when nobody participated?

We have a test to verify that we call the smart contract with an empty array of participants. The check `Object.keys(participants).length` keeps the test passing.

But is that the desired behaviour?

Reply:
> What about empty rounds just set the full score to the burner address? This way the IE marks the round as completed, yet not funds are wasted


SGTM. I wasn't sure if the smart contract would support that.

Opened a PR: https://github.com/filecoin-station/spark-evaluate/pull/56